src/ucp/tag/tag_send.c

```diff
@@ -363,10 +372,22 @@ UCS_PROFILE_FUNC(ucs_status_ptr_t, ucp_tag_send_nbx,
         cb = NULL;
     }
 
+    if (flags & UCP_EP_TAG_SEND_FLAG_EAGER) {
+        ucs_assert(!(flags & UCP_EP_TAG_SEND_FLAG_RNDV));
```

Comment:
already checked in line 324, no?

Reply:
> already checked in line 324, no?

yes, asserted in case if line 324 is removed...
removed assert